traffic_portal/app/src/common/modules/form/server/form.server.tpl.html

```diff
@@ -268,6 +253,15 @@ <h6>Deprecated</h6>
                         <label for="{{inf.name}}-max-bandwidth" ng-class="{'has-error': hasError(serverForm[inf.name+'-max-bandwidth'])}">Maximum Bandwidth</label>
                         <input id="{{inf.name}}-max-bandwidth" ng-model="inf.maxBandwidth" ng-class="{'has-error': hasError(serverForm[inf.name+'-max-bandwidth'])}" min="0" type="number" name="{{inf.name}}-max-bandwidth"/>
                         <small class="input-error" ng-show="hasPropertyError(serverForm[inf.name+'-max-bandwidth'], 'min')">Cannot be negative</small>
+                        <label for="{{inf.name}}-router-hostname" ng-class="{'has-error': hasError(serverForm[inf.name+'-router-hostname'])}">Router Hostname</label>
+                        <input id="{{inf.name}}-router-hostname" ng-model="inf.routerHostName" ng-class="{'has-error': hasError(serverForm[inf.name+'-router-hostname'])}" type="text" name="{{inf.name}}-router-hostname" maxlength="256" pattern="\S*"/>
+                        <small class="input-error" ng-show="hasPropertyError(serverForm[inf.name+'-router-hostname'], 'maxlength')">Too long</small>
+                        <small class="input-error" ng-show="hasPropertyError(serverForm[inf.name+'-router-hostname'], 'pattern')">No Spaces</small>
+                        <label for="{{inf.name}}-router-portname" ng-class="{'has-error': hasError(serverForm[inf.name+'-router-portname'])}">Router Port Name</label>
+                        <input id="{{inf.name}}-router-portname" ng-model="inf.routerPortName" ng-class="{'has-error': hasError(serverForm[inf.name+'-router-portname'])}" type="text" name="{{inf.name}}-router-portname" maxlength="256" pattern="\S*"/>
+                        <small class="input-error" ng-show="hasPropertyError(serverForm[inf.name+'-router-port'], 'maxlength')">Too long</small>
+                        <small class="input-error" ng-show="hasPropertyError(serverForm[inf.name+'-router-port'], 'pattern')">No Spaces</small>
```

Comment:
These errors aren't showing up when their conditions occur - I think it's because you're checking `inf.name+'router-port'` and the input's `id` is actually `{{inf.name}}-router-portname`

Reply:
aah yeah, copy paste error. Fixed now